### PR TITLE
Add a few files to the .hlxignore files

### PR DESCRIPTION
**Pull request by mokimo**
### Description
Add a few files to the .hlxignore file, which aren't interesting.

**Files that should be gone**
Root level .json files & codecov+the web test runner config
https://main--milo--mokimo.hlx.page/codecov.yaml
https://ignore-json-files--milo--mokimo.hlx.page/codecov.yaml

https://main--milo--mokimo.hlx.page/web-test-runner.config.mjs
https://ignore-json-files--milo--mokimo.hlx.page/web-test-runner.config.mjs

https://main--milo--mokimo.hlx.page/package.json
https://ignore-json-files--milo--mokimo.hlx.page/package.json

**Files that should still exist**
Any non-root level .json file
https://main--milo--adobecom.hlx.page/tools/sidekick/config.json
https://ignore-json-files--milo--mokimo.hlx.page/tools/sidekick/config.json

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.live/?martech=off
- After: https://ignore-json-files--milo--mokimo.hlx.live/?martech=off

### CODE REVIEW

**Comment by mokimo** [comment]
If we can think of any other files that we should exclude from getting pushed to the EDS pipeline/prod, we should add them too.

**Review by honstar — APPROVED** [review]

**Review by overmyheadandbody — APPROVED** [review]

**Review by sukamat — APPROVED** [review]

**Review by chrischrischris — APPROVED** [review]

**Review by narcis-radu — APPROVED** [review]

**Review by robert-bogos — APPROVED** [review]

**Review by SilviuLCF — APPROVED** [review]
Ready to STAGE deployment 
Testing details 
https://jira.corp.adobe.com/browse/MWPW-145752

**Comment by mokimo** [comment]
The aem-psi-check is correctly failing, as we don't want the assets to show up. We can safely ignore this check for this PR.

File: .hlxignore [hunk 1]
```diff
@@ -1,5 +1,9 @@
+*.json
+!*/**/*.json
```
Comment (yesil) [inline .hlxignore:2]: @mokimo  can you add `*.js.map` as guard against accidental commits?

**Review by yesil — APPROVED** [review]

**Pull request merged**